src/fidesops/core/config.py

```diff
@@ -178,14 +178,17 @@ class FidesopsConfig(FidesSettings):
     execution: ExecutionSettings
 
     is_test_mode: bool = os.getenv("TESTING") == "True"
-    hot_reloading: bool = os.getenv("DEV_MODE") == "True"
+    hot_reloading: bool = os.getenv("FIDESOPS_HOT_RELOAD") == "True"
+    dev_mode: bool = os.getenv("FIDESOPS_DEV_MODE") == "True"
 
     class Config:  # pylint: disable=C0115
         case_sensitive = True
 
-    logger.warning(f"Startup configuration: reloading = {hot_reloading}")
     logger.warning(
-        f'Startup configuration: pii logging = {os.getenv("LOG_PII") == "True"}'
+        f"Startup configuration: reloading = {hot_reloading}, dev_mode = {dev_mode}"
+    )
+    logger.warning(
+        f'Startup configuration: pii logging = {os.getenv("FIDESOPS_LOG_PII") == "True"}'
```

Comment:
Should these only warn under certain circumstances? For instance if log pii, dev mode or hot reloading is true? In theory if those aren't true, there's nothing to warn about

Reply:
I mean, up to you, I regularly verify this "Startup configuration", we could change it to be a logger.info?

Reply:
I mean, up to you, I regularly verify this "Startup configuration", we could change it to be a logger.info?